test-e2e/specs/presenceSpec.js

```diff
@@ -0,0 +1,95 @@
+/* global describe, it, expect, jasmine */

```

Comment:
Idea is so that we can share the same tests across multiple clients, and the framework takes care of setting up a cluster and everything for you so u can concentrate on just writing steps for whatever it is instead